src/internal.c

```diff
@@ -10132,8 +10132,8 @@ int CheckAvailableSize(WOLFSSL *ssl, int size)
 
 #ifdef WOLFSSL_DTLS13
 static int GetInputData(WOLFSSL *ssl, word32 size);
-static int GetDtls13RecordHeader(WOLFSSL* ssl, const byte* input,
-    word32* inOutIdx, RecordLayerHeader* rh, word16* size)
+static int GetDtls13RecordHeader(WOLFSSL* ssl, word32* inOutIdx,
+                                 RecordLayerHeader* rh, word16* size)
```

Comment:
formatting, on arguments overflowing, 4 spaces on the next line.

Reply:
indentation fixed now